deployments/liqo_chart/subcharts/discovery_chart/templates/unauth-role.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: anonymous-role-peering
+rules:
+  - apiGroups: ["discovery.liqo.io"]
+    resources: ["peeringrequests"]
+    verbs: ["get", "create", "delete", "update"]
```

Comment:
I am a bit scaried about all those privileges. Can't we use just create?

Reply:
At the moment not, because, we need get to check if peering request exsists, update to set the reference to the created secret (with kubeconfig credentials) and delete to delete it when user wants to disjoin clusters